src/python/tests/core/crash_analysis/stack_parsing/stack_analyzer_test.py

```diff
@@ -1321,6 +1321,18 @@ def test_asan_double_free(self):
                                   expected_state, expected_stacktrace,
                                   expected_security_flag)
 
+  def test_fuchsia_asan(self):
+    """Test for Fuchsia ASan crashes."""
+    data = self._read_test_data('fuchsia_asan.txt')
+    expected_type = 'Heap-buffer-overflow'
+    expected_state = 'NULL'
```

Comment:
to test this properly, can you get a crash with more frames above LLVMFuzzerTestOneInput? The reason this is NULL is because we ignore all the frames in the stack.